Makefile

```diff
@@ -73,6 +73,7 @@ str_match_test:
 # testing
 ##
 test: joy joy_test.py
+	./download_test_pcaps.sh test_pcaps/pcapr
```

Comment:
Sounds good. Those caps are static and all pretty small, so it shouldn't be an issue including them. The main motivation for having them downloaded separately was to allow the script to download larger caps in the future that we wouldn't want to include in the source distribution.